mods/cnc/maps/n64gdi01/map.yaml

```diff
@@ -0,0 +1,813 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: N64 Special Ops - GDI 1
```

Comment:
N64 is also a workmark so try to avoid it as well. https://trademarks.justia.com/750/77/n64-75077622.html